src/test/java/pages/SubmitNewLanguagePage.java

```diff
@@ -1,10 +1,30 @@
 package pages;
 
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
 
 public class SubmitNewLanguagePage extends SubmitNewLanguageSubmenuPage{
 
-    public SubmitNewLanguagePage(WebDriver driver) {
+    @FindBy(xpath = "//p/input[@name='submitlanguage']")
+    private WebElement goButton;
+
+    @FindBy(xpath = "//div[@id='main']/p[@style]")
+    private WebElement errorMessage;
+
+    public SubmitNewLanguagePage(WebDriver driver)
+    {
```

Comment:
Лучше использовать единый стиль 
```suggestion
    public SubmitNewLanguagePage(WebDriver driver) {
```